### ISSUE TITLE
Gamepad bindings

### ISSUE THREAD

**Issue by johnarnfield** [mynameisgonz/FFXI-Apps#1]
This is fantastic. Thanks so much for sharing. 

Do you have any idea if it would be possible to add bindings for gamepads?

**Comment by mynameisgonz** [comment]
I will look into what can be done.

**Comment by Noobcakes** [comment]
I agree. This looks dope. I have a couple feature suggestions as well. 
*  Cycle forward/back through instances of ffxi
*  Ability to set the order it cycles through instances of ffxi
*  Ability to remove/add instances from the cycle order (useful when you have idle characters open and don't want to tab through them)

I've been running wincontrol using xinput to map the forward/back functions to my xbox one controller and it works, but I'd much rather use something that has been updated in the last 15 years. Thanks for making something already so useful.
